src/Mod/Draft/draftobjects/array.py

```diff
@@ -75,190 +75,201 @@ def set_general_properties(self, obj):
         properties = obj.PropertiesList
 
         if "Base" not in properties:
-            _tip = "The base object that will be duplicated"
-            obj.addProperty("App::PropertyLink",
-                            "Base",
-                            "Objects",
-                            QT_TRANSLATE_NOOP("App::Property", _tip))
+            _tip = QT_TRANSLATE_NOOP("App::Property", "The base object that will be duplicated")
+            obj.addProperty("App::PropertyLink", "Base", "Objects", _tip)
             obj.Base = None
 
         if "ArrayType" not in properties:
-            _tip = ("The type of array to create.\n"
+            _tip = QT_TRANSLATE_NOOP("App::Property", 
+                    ("The type of array to create.\n"
```

Comment:
You don't need to add another set of parentheses. The parentheses in `_NOOP( ... )` are enough for line continuation.